subsys/input/CMakeLists.txt

```diff
@@ -6,3 +6,12 @@ zephyr_library_sources(input.c)
 zephyr_library_sources(input_utils.c)
 
 zephyr_library_sources_ifdef(CONFIG_INPUT_LONGPRESS input_longpress.c)
+
+if(CONFIG_INPUT_LVGL)
+
+zephyr_include_directories(${ZEPHYR_BASE}/subsys/input)
```

Comment:
I don't think this is needed since the common file is include with quotes (`#include "input_lvgl_common.h"`) and the files that use it are in the same directory.